osu.Game/Screens/Play/PlayerLoader.cs

```diff
@@ -56,6 +59,17 @@ public class PlayerLoader : ScreenWithBeatmapBackground
 
         private IdleTracker idleTracker;
 
+        [Resolved]
+        private NotificationOverlay notificationOverlay { get; set; }
+
+        [Resolved]
+        private VolumeOverlay volumeOverlay { get; set; }
+
+        [Resolved]
+        private AudioManager audioManager { get; set; }
+
+        private static bool muteWarningShownOnce;
```

Comment:
You can use #6216 for this instead.

Reply:
Kinda conflicts what peppy said to me :shrug:
> ppy 09/16/2019
really these kinds of things should be stored somewhere more central, but we don't have anything for that yet
something like ConfigManager which doesn't save to disk
`class SessionStatics : ConfigManager` (cached in `GameBase`) could work well, but it's out of scope of what you're implementing

Reply:
How does it conflict?

Reply:
Then it does not, I recalled hearing something like SessionStatics somewhere and it not fitting what I want to do.

Reply:
just a dependency you can use to save values for the whole session of the game
```csharp
[BackgroundDependencyLoader]
private void load(SessionStatics sessionStatics)
{
    if (!sessionStatics.Get(Statics.UsefulValue))
        // do something once.

    sessionStatics.Set(Statics.UsefulValue, true);
}